diff --git a/src/webview/mod.rs b/src/webview/mod.rs
index b89d3c9..c38e0b6 100644
--- a/src/webview/mod.rs
+++ b/src/webview/mod.rs
@@ -7,0 +8 @@ pub(crate) mod plugin;
+mod headless;
@@ -1301 +1302 @@ pub struct Webview<R: Runtime> {
-  pub(crate) webview: DetachedWebview<EventLoopMessage, R>,
+  pub(crate) webview: headless::ManagedWebview<R>,
@@ -1359 +1360 @@ impl<R: Runtime> Webview<R> {
-      webview,
+      webview: webview.into(),
